app/model/media/AudioModel.js

```diff
@@ -260,73 +260,109 @@ SDL.AudioModel = Em.Object.extend({
     /**
      * Turn on CD
      */
-    turnOnCD: function () {
+    turnOnCD: function (event=null, is_background=false) {
```

Comment:
```suggestion
    turnOnCD: function (is_background=false) {
```
Doesn't look like this `event` parameter is used?

Reply:
Not when the source is switched through an RPC request but when the "Go To CD" button is pressed (https://github.com/smartdevicelink/sdl_hmi/blob/master/app/view/infoView.js#L175) a DOM/Ember event is passed through to turnOnCD. This would overwrite the is_background with an object rather than a bool
![Screenshot from 2022-09-21 11-13-44](https://user-images.githubusercontent.com/10703941/191576179-19899467-45a2-4dc2-aa00-77c35d63bdf7.png)